src/Login/LoginForm.tsx

```diff
@@ -56,6 +57,12 @@ export const LoginForm: React.FC<LoginFormProps> = () => {
                 Cookie.set(CookieEnum.USERID, resp.data.data.userId);
                 Cookie.set(CookieEnum.USERNAME, `${resp.data.data.firstName} ${resp.data.data.lastName}`);
 
+                LogRocket.identify('qdgaa4/rederly', {
+                    name: `${resp.data.data.firstName} ${resp.data.data.lastName}`,
```

Comment:
@doombot-exe do we have to worry about FERPA or private info here at all, I don't think this is an issue

Reply:
Talking about this now actually